core/ledger/ledger_interface.go

```diff
@@ -470,10 +470,42 @@ func (filter PvtNsCollFilter) Has(ns string, coll string) bool {
 	return collFilter[coll]
 }
 
+// PrivateReads captures which private data collections are read during TX simulation.
+type PrivateReads map[string]map[string]bool
```

Comment:
Not a big deal but better to use canonical type `map[string]map[string]struct{}` for a set.